raiden-dapp/src/components/UpdateSnackbar.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <span v-if="updateAvailable">
+    <blurred-overlay :show="updateAvailable" />
+    <v-snackbar v-model="updateAvailable" :timeout="0" color="primary">
+      {{ $t('update.available') }}
+      <v-btn dark text :loading="isUpdating" @click="update">
+        {{ $t('update.update') }}
+      </v-btn>
+    </v-snackbar>
+  </span>
+</template>
+
+<script lang="ts">
+/* istanbul ignore file */
```

Comment:
I've excluded this from code coverage, because it's mostly using Web API and testing it would be pretty difficult and would yield very little insight for us IMO

Reply:
It makes absolute sense. Thanks for doing that.